packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -497,6 +499,7 @@ class NextjsComponent extends Component {
     delete defaultLambdaAtEdgeConfig["origin-response"];
 
     const cloudFrontOutputs = await cloudFront({
+      distributionId: cloudFrontDistributionId || null,
```

Comment:
Is the or to null necessary? 

```suggestion
      distributionId: cloudFrontDistributionId,
```

Reply:
done